### PR TITLE
feat: pull model yaml from hf

### PR DESCRIPTION
**Pull request by namchuai**
## Describe Your Changes

- allow user to pull model from janhq from HuggingFace

## Videos
https://github.com/janhq/cortex/assets/10397206/70f7af9e-3c86-4dbd-8322-91cf43a44987

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: cortex-js/src/infrastructure/commanders/models/model-pull.command.ts [hunk 1]
```diff
@@ -1,25 +1,151 @@
...
+            console.error('Invalid version');
+            rl.prompt();
+          }
+        } catch (e) {
+          console.error('Invalid version');
+          rl.prompt();
+        }
+      });
+    });
+  }
```
Comment (louis-jan) [inline cortex-js/src/infrastructure/commanders/models/model-pull.command.ts:105]: Why not use Choices? It's a lot shorter

File: cortex-js/src/infrastructure/commanders/usecases/models.cli.usecases.ts [hunk 2]
```diff
@@ -98,6 +98,41 @@ export class ModelsCliUsecases {
...
+      settings: {
+        prompt_template: '', // TODO: get prompt template from the file
+      },
+      parameters: {
+        stop: [],
+      },
+      metadata: {
+        author: 'janhq', // TODO: get author from the file
+        size: 0, // TODO: get size from the file
+        tags: [],
```
Comment (louis-jan) [inline cortex-js/src/infrastructure/commanders/usecases/models.cli.usecases.ts:122]: Please address all the TODOs

File: cortex-js/src/infrastructure/commanders/models/model-pull.command.ts (hunk 1 above)
Comment (namchuai) [reply cortex-js/src/infrastructure/commanders/models/model-pull.command.ts:105]: ah yes, I forgot about the `inquirerService`

**Comment by louis-jan** [comment]
How can I make it work?

Screenshot 2024-05-24 at 10 31 57

**Review by louis-jan — APPROVED** [review]
LGTM

**Pull request merged**